### PR TITLE
test: change s3-tests from single bucket to double buckets

### PR DESCRIPTION
**Pull request by AragornBUPT**
We found a bug when multi bucket were operated.
The s3-tests case now only support single bucket tests.
So I add a bucket named 's3test'.

Signed-off-by: wanglei469 <wanglei469@ke.com>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -355,7 +355,7 @@ services:
             - ./script/run_test.sh:/cfs/script/start.sh
             - ./script/start_client.sh:/cfs/script/start_client.sh
             - ./ltp/runtest/fs:/opt/ltp/runtest/fs
-            - ./s3tests:/opt/s3tests:ro
+            - ./s3tests:/opt/s3tests
```
Comment (shyodx) [inline docker/docker-compose.yml:358]: 最好能保留 ro 属性，避免侵入式修改 host 上的文件

File: docker/script/run_test.sh [hunk 2]
```diff
@@ -260,22 +278,24 @@ run_s3_test() {
     fi
     echo -e "\033[32mdone\033[0m"
 
-    python3 -m unittest2 discover ${work_path} "*.py" -v
-    if [[ $? -ne 0 ]]; then
-        exit 1
-    fi
+    sed -i "s/BUCKET = 's3test'/BUCKET = 'ltptest'/g" ${work_path}/env.py
```
Comment (shyodx) [inline docker/script/run_test.sh:281]: An argument or an environment variable could be considered to pass to unittest

File: docker/script/run_test.sh [hunk 3]
```diff
@@ -221,15 +223,30 @@ stop_client() {
     umount ${MntPoint} && echo -e "\033[32mdone\033[0m" || { echo -e "\033[31mfail\033[0m"; exit 1; }
 }
 
-delete_volume() {
-    echo -n "Deleting volume   ... "
-    ${cli} volume delete ${VolName} -y &> /dev/null
-    if [[ $? -eq 0 ]]; then
-        echo -e "\033[32mdone\033[0m"
-        return
+delete_volumes() {
```
Comment (shyodx) [inline docker/script/run_test.sh:226]: I think the `delete_volumes` function could be simplified as following:
```shell
for volName in ${VolNameList[@]}; do
    echo -n "Deleting volume $volName ..."
    ${cli} volume delete $volName -y &> /dev/null
    if [[ $? -eq 0 ]]; then
        echo -e "\033[32mdone\033[0m"
        return
    fi
    echo -e "\033[31mfail\033[0m"
    exit 1
done
```

File: docker/script/run_test.sh [hunk 4]
```diff
@@ -113,14 +115,14 @@ show_cluster_info() {
     echo &>> ${tmp_file}
     ${cli} user info ${Owner} &>> ${tmp_file}
     echo &>> ${tmp_file}
-    ${cli} volume info ${VolName} &>> ${tmp_file}
+    ${cli} volume info ${VolNameList[0]} &>> ${tmp_file}
```
Comment (shyodx) [inline docker/script/run_test.sh:118]: Do we need to show info of `VolNameList[1]`? How about:
```shell
for VolName in ${VolNameList[@]}; do
    ${cli} volume info ${VolName} &>> ${tmp_file}
done
```

File: docker/script/run_test.sh [hunk 5]
```diff
@@ -113,14 +115,14 @@ show_cluster_info() {
...
     echo &>> ${tmp_file}
     cat /tmp/collect_cluster_info | grep -v "Master address"
 }
 
 add_data_partitions() {
     echo -n "Increasing DPs    ... "
-    ${cli} vol add-dp ${VolName} 20 &> /dev/null
+    ${cli} vol add-dp ${VolNameList[0]} 20 &> /dev/null
```
Comment (shyodx) [inline docker/script/run_test.sh:125]: ditto.

**Review by shyodx — APPROVED** [review]

**Comment by leonrayang** [comment]
https://github.com/cubeFS/cubefs/issues/1417

**Pull request merged**